packages/genesys-spark-components/src/components/stable/gux-listbox/gux-listbox.service.ts

```diff
@@ -26,14 +28,21 @@ function getActiveOption(list: HTMLGuxListboxElement): ListboxOptionElement {
   return getListOptions(list).find(option => option.active);
 }
 
+function isNonSelectableOption(option: HTMLElement): boolean {
+  return !validOptionTagNames.includes(option.tagName);
+}
+
```

Comment:
I think this would be simpler if you use an `isSelectableOption` function and include the disabled and filtered check in it.

```javascript
function isSelectableOption(option: HTMLElement): boolean {
  return validOptionTagNames.includes(option.tagName) && (!option.disabled || !option.filtered);
}
```

Then you would have calls like 

```javascript
const active = (option === element) && isSelectableOption(option) ;
```

and 

```javascript
while (firstOption && !isSelectableOption(firstOption)) {
  firstOption = firstOption.nextElementSibling as ListboxOptionElement;
}
```

Reply:
Good idea--I've updated to this with the modification that the option should be both enabled and non-filtered to be selectable, which I think was just an error in your quick example.

Reply:
🙈